people/people_service.go

```diff
@@ -225,6 +228,14 @@ func (s service) Count() (int, error) {
 	return results[0].Count, nil
 }
 
+func randSeq(n int) string {
```

Comment:
its for instances where a person has two tme identifiers. the variables for each identifier need to be different otherwise the cypher fails. You are right it will fail 1 in 27000 times where the same sequence is generated twice in a row but i couldnt find a better way to solve this problem. I asked Galia and in orgs she set up a seperate function so I could do that if we would prefer.